presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/ActualProperties.java

```diff
@@ -125,22 +125,18 @@ public boolean isNodePartitionedOn(Collection<VariableReferenceExpression> colum
         return global.isNodePartitionedOn(columns, constants.keySet(), nullsAndAnyReplicated);
     }
 
-    @Deprecated
-    public boolean isCompatibleTablePartitioningWith(Partitioning partitioning, boolean nullsAndAnyReplicated, Metadata metadata, Session session)
+    public boolean isNodePartitionedOn(Partitioning partitioning, boolean nullsAndAnyReplicated)
```

Comment:
To reviewer: this part sort of reverts https://github.com/prestodb/presto/pull/11749/commits/c223aece743a391fae577a555d438422aa1e3367#diff-e38c0df6e5590ddbaa463fd73d348910R400